resources/templates/base.twig

```diff
@@ -34,6 +33,23 @@
             'lang': {'publish': '{{ lang('publish') }}', 'hide': '{{ lang('hide') }}', 'dropzone': '{{ lang('drop_to_upload') }}'}
         }
     </script>
+    <meta id="embed-title" property="og:site_name" content="{{ config.app_name }}">
+    <meta id="embed-url" property="og:url" content="{{ url }}">
+    <!-- This will only display an image, but that's the most important for now -->
+    {% if type %}
+        <meta name="twitter:card" content="summary_large_image">
+        <meta property="og:type" content="website" />
+        <meta id="embed-title" property="og:title" content="{{ media.filename }} ({{ media.size }})">
+        <meta id="embed-image" property="og:image" content="{{ url }}/raw">
+        <meta id="embed-desc" property="og:description" content="Uploaded at {{ media.timestamp }} by {{ media.username }}">
```

Comment:
This needs to be translatable.

Reply:
As description I would keep only the timestamp, and maybe the list of the tags. I wouldn't expose the internal user name.